code/game/objects/items/handcuffs.dm

```diff
@@ -59,8 +59,9 @@
 
 	if(!C.handcuffed)
 		if(C.canBeHandcuffed())
-			C.visible_message(span_danger("[user] is trying to put [src.name] on [C]!"), \
-								span_userdanger("[user] is trying to put [src.name] on you!"))
+			C.visible_message(span_danger("[user] is trying to put [name] on [C]!"), \
+								span_userdanger("[user] is trying to put [name] on you!"), \
+								span_userdanger("You feel someone grab your wrists, the clink of [name] rattling around your ears!"))
```

Comment:
are you sure the third message isnt for everyone blind nearby?

Reply:
Oh, fikou's right. Guess you'll have to do something in this style:
```
if(C.isblind())
	C.show_message(span_userdanger("You feel someone grab your wrists, the clink of [name] rattling around your ears!"), MSG_AUDIBLE)
```

Reply:
Ah, I'll do that in a moment